src/templates/QuickstartDetails.js

```diff
@@ -49,6 +50,16 @@ const QuickstartDetails = ({ data, location }) => {
   const pack = data.quickstarts;
   const packUrl = pack.packUrl || QUICKSTARTS_REPO;
   const tessen = useTessen();
+  const breadcrumbSegments = [
+    {
+      name: 'Instant Observability (I/O)',
+      url: '/instant-observability/',
+    },
+    {
+      name: pack.name,
+      url: null,
```

Comment:
I left a comment about this in the other file. Even if we decide we _don't_ want to account for this in the component, I think it might be better to just not supply this value. `undefined` and `null` will both evaluate to `false` and I think it's a little cleaner this way:

```suggestion
```

Reply:
oh right. thanks!